src/functions/Coverage.ps1

```diff
@@ -87,7 +88,7 @@ function Exit-CoverageAnalysis {
     # to only get those that are not $null
     # (like if we did $breakpoints | where {$_ -ne $null})
     # so DON'T change this.
-    $breakpoints = @($CommandCoverage.Breakpoint) -ne $null
+    $breakpoints = null -ne @($CommandCoverage.Breakpoint)
```

Comment:
```suggestion
    $breakpoints = $null -ne @($CommandCoverage.Breakpoint)
```

Reply:
Did you read the comment above this line?